packages/p2p/src/components/my-ads/my-ads-table.jsx

```diff
@@ -41,57 +42,72 @@ const MyAdsTable = observer(() => {
                         'p2p-my-ads__table--disabled': !general_store.is_listed,
                     })}
                 >
-                    <Table.Header>
-                        <Table.Row className='p2p-my-ads__table-row'>
-                            {getHeaders(general_store.client.currency).map(header => (
-                                <Table.Head key={header.text}>{header.text}</Table.Head>
-                            ))}
-                        </Table.Row>
-                    </Table.Header>
-                    <Table.Body className='p2p-my-ads__table-body'>
-                        <InfiniteDataList
-                            data_list_className='p2p-my-ads__data-list'
-                            items={my_ads_store.adverts}
-                            rowRenderer={row_props => <MyAdsRowRenderer {...row_props} />}
-                            has_more_items_to_load={my_ads_store.has_more_items_to_load}
-                            loadMoreRowsFn={my_ads_store.loadMoreAds}
-                            keyMapperFn={item => item.id}
-                        />
-                    </Table.Body>
+                    {isDesktop() && (
+                        <Table.Header>
+                            <Table.Row className='p2p-my-ads__table-row'>
+                                {getHeaders(general_store.client.currency).map(header => (
+                                    <Table.Head key={header.text}>{header.text}</Table.Head>
+                                ))}
+                            </Table.Row>
+                        </Table.Header>
+                    )}
+
+                    {isMobile() ? (
+                        <Div100vhContainer height_offset='26rem'>
+                            <InfiniteDataList
+                                data_list_className='p2p-my-ads__data-list'
+                                items={my_ads_store.adverts}
+                                rowRenderer={row_props => <MyAdsRowRenderer {...row_props} />}
+                                has_more_items_to_load={my_ads_store.has_more_items_to_load}
+                                loadMoreRowsFn={my_ads_store.loadMoreAds}
+                                keyMapperFn={item => item.id}
+                            />
+                        </Div100vhContainer>
+                    ) : (
+                        <Table.Body>
+                            <InfiniteDataList
+                                data_list_className='p2p-my-ads__data-list'
+                                items={my_ads_store.adverts}
+                                rowRenderer={row_props => <MyAdsRowRenderer {...row_props} />}
+                                has_more_items_to_load={my_ads_store.has_more_items_to_load}
+                                loadMoreRowsFn={my_ads_store.loadMoreAds}
+                                keyMapperFn={item => item.id}
+                            />
+                        </Table.Body>
+                    )}
```

Comment:
This was repeated because our infinite loader list was a little messy but now that Aaron has fixed it (thanks @aaron-binary 🙏 ), we can just keep the `InfiniteDataList` in the body.